benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+
+##########################################################
+# Google benchmark
+
+ExternalProject_Add(
+    google_benchmark
```

Comment:
How is google_benchmark different from follys, any advantages? Would be nice to not add another dep if possible.

https://github.com/facebook/folly/blob/master/folly/docs/Benchmark.md